insolar/bus/bus.go

```diff
@@ -86,6 +68,8 @@ type Sender interface {
 	SendTarget(ctx context.Context, msg *message.Message, target insolar.Reference) (<-chan *message.Message, func())
 	// Reply sends message in response to another message.
 	Reply(ctx context.Context, origin payload.Meta, reply *message.Message)
+
+	IncomingMessageRouter(handle message.HandlerFunc) message.HandlerFunc
```

Comment:
it is moved to interface, so I can use Sender as incoming router, instead of having both sender and *bus.Bus

Reply:
where do you need both?

Reply:
in contractrtequester

Reply:
I think better solution will be to separate it in different interface and create different mock.

So it could be injected, even though it will be the same bus.Bus

Reply:
removed this stuff, all handlers now live in components initialization